DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: ReactomeContentService4R
 Title: R Interface for the Reactome Content Service
-Version: 0.1.0
+Version: 0.99.0
```

Comment:
Ok, this looks weird. Please explain?

Reply:
Hmm this might actually be unnecessary. @chilampoon could you share where you saw this version requirement? Only thing I can find is here, which is just an example of a version: https://github.com/Bioconductor/Contributions#what-to-expect

Reply:
New packages submitted to Bioconductor should set Version: 0.99.0 in the DESCRIPTION file. :/

https://bioconductor.org/developers/how-to/version-numbering/

Reply:
Ok.